client/state/jetpack-connection-health/actions.js

```diff
@@ -37,3 +37,66 @@ export const setJetpackConnectionUnhealthy = ( siteId, errorCode ) => ( {
 	siteId,
 	errorCode,
 } );
+
+/**
+ * Requests the Jetpack connection status from the server
+ *
+ * This is called when the Jetpack connection is maybe unhealthy and we want to confirm
+ * the status by calling the health status API.
+ * We also throttle the requests to avoid calling the API too often.
+ * @param {number} siteId The site id to which the status belongs
+ * @returns {Function} Action thunk
+ */
+export const requestJetpackConnectionHealthStatus = ( siteId ) => ( dispatch, getState ) => {
+	const currentState = getState();
+	const lastRequestTime = currentState.jetpackConnectionHealth[ siteId ]?.lastRequestTime;
+	if ( lastRequestTime && Date.now() - lastRequestTime < STALE_CONNECTION_HEALTH_THRESHOLD ) {
```

Comment:
I find it unexpected that I call a `requestJetpackConnectionHealthStatus()` action, but it won't request the connection status if it has recently been requested (AKA throttling). So, should this limitation be abstracted somewhere else, outside the data layer, allowing us to request fresh health status if we want to?